src/docs/cursor-ops.dox

```diff
@@ -103,6 +103,17 @@ fail with ::WT_DUPLICATE_KEY if the record previously exists, and
 WT_CURSOR::update and WT_CURSOR::remove will fail with ::WT_NOTFOUND if the
 record does not previously exist.
 
+@snippet ex_cursor.c cursor largest key
+
+The WT_SESSION::largest_key \c gets the largest key in a table regardless of
+visibility. Therefore, it can return a key that may have been already deleted.
+
```

Comment:
I think this is true for any call, that is, `cursor.next` can return a key that's "already been deleted". I'd remove that phrase.